app/src/main/java/hr/foi/air/servicesync/ui/contents/SearchContent.kt

```diff
@@ -1,23 +1,369 @@
 import androidx.compose.foundation.layout.fillMaxSize
 import androidx.compose.foundation.layout.fillMaxWidth
-import androidx.compose.material3.Button
-import androidx.compose.material3.Scaffold
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.heightIn
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.wrapContentSize
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.LazyRow
+import androidx.compose.foundation.lazy.items
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.shape.AbsoluteRoundedCornerShape
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.ArrowDropDown
+import androidx.compose.material.icons.filled.Search
+import androidx.compose.material3.Card
+import androidx.compose.material3.CardColors
+import androidx.compose.material3.CircularProgressIndicator
+import androidx.compose.material3.DropdownMenu
+import androidx.compose.material3.DropdownMenuItem
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.Icon
+import androidx.compose.material3.MaterialTheme
 import androidx.compose.material3.Text
+import androidx.compose.material3.TextField
+import androidx.compose.material3.TextFieldDefaults
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.input.pointer.pointerInput
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.input.TextFieldValue
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.zIndex
+import androidx.navigation.NavController
+import com.example.compose.onSurfaceDark
+import com.example.compose.onSurfaceLight
+import com.example.compose.onSurfaceVariantDark
+import com.example.compose.onSurfaceVariantLight
+import com.example.compose.outlineDark
+import com.example.compose.outlineLight
+import com.example.compose.surfaceContainerHighDark
+import com.example.compose.surfaceContainerHighLight
+import com.example.compose.surfaceContainerLowDark
+import com.example.compose.surfaceContainerLowLight
+import com.google.firebase.firestore.FirebaseFirestore
+import hr.foi.air.servicesync.ui.components.CompanyCard
+import hr.foi.air.servicesync.ui.components.isDark
+import java.text.Collator
+import java.util.Locale
 
+@ExperimentalMaterial3Api
 @Composable
-fun SearchContent(modifier: Modifier = Modifier)
+fun SearchContent(modifier: Modifier = Modifier, navController: NavController)
```

Comment:
Okay. This contains EVERYTHING. This should be refactored in a later sprint in a way that everything is separated in separate layers.